source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,95 @@
+.. include:: <isonum.txt>
+
+How to Ask Effective Questions
+==============================
+
+Asking questions effectively is one of the most important skills to develop, especially for your life outside of robotics.
+Being able to learn from your questions properly is crucial and as such, you should be able to ask good questions, help those who want to help you, and make the most of what you are taught.
+
+Research First
+--------------
+
+Before you ask a question, you should do your own research to find the answer.
+This will take more time and more effort, but it is crucial to learn to find solutions on your own, and learn how to research effectively to find solutions to your problems.
+In the end, this research will pay off in the long run, helping you to learn about topics relevant to your question, and often helping you to understand the roots of the problems you encounter.
+
+Here are some steps that you could take to research your problem:
+
+- **Checking Official Documentation** - If documentation or instructions are available, make sure to carefully examine the relevant sections of them.
+  Your problem may already be addressed, or may be caused by you misreading something.
+
+- **Google** - Simply search your problem.
+  You will be surprised by how often this solves the issue.
+  Try word the question in many different ways to find similar questions that may have already been asked.
+  Focus on including keywords about your problem and if it is about programming, include relevant error messages and codes.
+
+.. tip::
+    When looking for past documents or sections of textbooks, add "filetype:pdf" to the beginning of your search to filter the search results into only .pdf documents.
+
+- **YouTube** - There are many great resources on YouTube which can help with even obscure problems.
+
+- **Reddit and Discord** - The `FTC Subreddit <https://www.reddit.com/r/ftc/>`_ and `Discord Server <https://discord.gg/first-tech-challenge>`_ can also be useful places to find previous answers or to ask your own.
```

Comment:
We link to and correspondingly warn about the FTC Discord elsewhere (https://gm0.org/en/latest/docs/useful-resources.html), which I think is important. it might be useful to either take this warning or to link to that reference instead of the server directly.

Reply:
Fixed in most recent commit 
[https://github.com/gamemanual0/gm0/pull/432/commits/c2f3ae902bb6f276fcf27a23e403ce33948c3e9b](url)